tracer/test/snapshots/NetActivitySdkTests.verified.txt

```diff
@@ -0,0 +1,456 @@
+[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: Samples.NetActivitySdk.server,
+    Resource: Ctor1,
+    Service: Samples.NetActivitySdk,
+    Type: web,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      otel.library.name: Samples.NetActivitySdk,
+      otel.status_code: STATUS_CODE_UNSET,
+      otel.trace_id: Guid_1,
+      runtime-id: Guid_2,
+      span.kind: server,
+      _dd.p.dm: -0
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.agent_psr: 1.0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_1,
```

Comment:
I'm surprised that this has the same TraceId as the span above since this was created with the constructor that accepts an `ActivityContext` object. Do you think this is something we need to fix in the Tracer?